### DISCUSSION TITLE
Computing Metrics

### DISCUSSION THREAD

**Discussion by ojasvijain** [tbepler/topaz#196] [General]
Hi!

I wanted to compute metrics such as precision, recall, ap etc. based on the code provided on https://github.com/tbepler/topaz/blob/master/topaz/metrics.py. 

I just wanted to understand what the "target" and "pred" looks like in the average_prediction function. I have 2 files (for each image) which consist of the particle and the corresponding bounding box coordinates (4 points). Is it that what is expected?

Thanks!

**Comment by tbepler** [comment]
The code you're referring to expects target labels (1 for positive, 0 for negative) as the target and a corresponding array of prediction scores from the model as the pred array.
